### PR TITLE
ADD SEARCHER text and open office hours

### PR DESCRIPTION
**Pull request by Vedanth-Ramji**

### CODE REVIEW

**Comment by luispedro** [comment]
@cocodyq: can you fix the deployment error?

**Comment by Vedanth-Ramji** [comment]
I found this issue: https://github.com/ryan-haskell/date-format/issues/40#issue-2183844717.

I tried `elm-json upgrade` but that didn't resolve the problem for me.

**Comment by luispedro** [comment]
Can you rebase this to the current master to generate the preview?

File: content/open-office-hours.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Open Office Hours with Dr. Luis Pedro Coelho"
+meta: "Big Data Biology Lab Office Hours"
+---
+
+[Dr. Luis Pedro Coelho](/people/luis_pedro_coelho) will hold two Zoom open office hour sessions on:
+
+- [April 24, 2024 08:30am AEST (GMT+10.00)](https://qut.zoom.us/meeting/register/tZcsfuCopjwoHdBhmkJQY9mzd0Zu_LYCxrcv#/registration)
```
Comment (luispedro) [inline content/open-office-hours.md:8]: Can you add some info on what the time is in India/Berlin/New York/LA? or a link to the right https://everytimezone.com/ time

**Review by luispedro — CHANGES_REQUESTED** [review]

**Pull request merged**